From: SeongJae Park <sj@kernel.org>
Subject: Docs/mm/damon/design: fix build warning
Date: Tue, 7 May 2024 09:17:47 -0700

Commit b7138c7d40b0 ("Docs/mm/damon/design: use a list for supported
filters") of mm-unstable tree is causing below warning and error with
'make htmldocs'.

    Documentation/mm/damon/design.rst:482: ERROR: Unexpected indentation.
    Documentation/mm/damon/design.rst:483: WARNING: Block quote ends without a blank line; unexpected unindent.

The problem caused by wrong indentation for nested list items.  Fix the
wrong indentation.

Link: https://lkml.kernel.org/r/20240507161747.52430-1-sj@kernel.org
Fixes: b7138c7d40b0 ("Docs/mm/damon/design: use a list for supported filters")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lore.kernel.org/20240507162623.4d94d455@canb.auug.org.au
Signed-off-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/mm/damon/design.rst |   22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

--- a/Documentation/mm/damon/design.rst~docs-mm-damon-design-use-a-list-for-supported-filters-fix
+++ a/Documentation/mm/damon/design.rst
@@ -472,21 +472,21 @@ counted as the scheme has tried.  This d
 Below types of filters are currently supported.
 
 - anonymous page
-  - Applied to pages that containing data that not stored in files.
-  - Handled by operations set layer.  Supported by only ``paddr`` set.
+    - Applied to pages that containing data that not stored in files.
+    - Handled by operations set layer.  Supported by only ``paddr`` set.
 - memory cgroup
-  - Applied to pages that belonging to a given cgroup.
-  - Handled by operations set layer.  Supported by only ``paddr`` set.
+    - Applied to pages that belonging to a given cgroup.
+    - Handled by operations set layer.  Supported by only ``paddr`` set.
 - young page
-  - Applied to pages that are accessed after the last access check from the
-    scheme.
-  - Handled by operations set layer.  Supported by only ``paddr`` set.
+    - Applied to pages that are accessed after the last access check from the
+      scheme.
+    - Handled by operations set layer.  Supported by only ``paddr`` set.
 - address range
-  - Applied to pages that belonging to a given address range.
-  - Handled by the core logic.
+    - Applied to pages that belonging to a given address range.
+    - Handled by the core logic.
 - DAMON monitoring target
-  - Applied to pages that belonging to a given DAMON monitoring target.
-  - Handled by the core logic.
+    - Applied to pages that belonging to a given DAMON monitoring target.
+    - Handled by the core logic.
 
 
 Application Programming Interface
_
